<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>huobi</class>
 <widget class="QWidget" name="huobi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>548</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>10000</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
    <pointsize>-1</pointsize>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>拉胯计算器</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-size:20px;</string>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-20</y>
     <width>411</width>
     <height>571</height>
    </rect>
   </property>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>CNY</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EUR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GBP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>JPY</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CAD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>USR</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1 CNY = 0.1572 USD</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>61</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="maximum">
     <double>999999999.990000009536743</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
font:50px &quot;宋体&quot;;


}</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="maximum">
     <double>999999999.990000009536743</double>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>CNY</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EUR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GBP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>JPY</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CAD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>USR</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1 CNY = 0.1572 USD</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
